### DISCUSSION TITLE
Mute alert sounds

### DISCUSSION THREAD

**Discussion by xheisenbugx** [zed-industries/zed#19052] [Feature Requests]
Hi everyone, i am using the following keymap configuration in mac:
```
  {
    "context": "Workspace", // navigation
    "bindings": {
      "ctrl-/": "terminal_panel::ToggleFocus" // zed specific
    }
  },
```

I've noticed that this keymap triggers an annoying alert sound on macOS, which doesn’t happen when using the default keymap (`Ctrl + ``). It seems to be related to the integrated terminal, as this issue also occurs in other terminal emulators like WezTerm. However, in iTerm2, instead of emitting the sound, it displays a bell icon, as shown here:

<img width="233" alt="Captura de pantalla 2024-10-10 a la(s) 8 15 39 p m" src="https://github.com/user-attachments/assets/e6495fc6-799b-4adc-ba6b-ed802c22b2e8">

Is it possible to have a configuration in Zed to avoid alert sounds?

I would appreciate your help, thanks in advance!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
